src/widgets/qownnotesmarkdowntextedit.cpp

```diff
@@ -72,6 +72,13 @@ QOwnNotesMarkdownTextEdit::QOwnNotesMarkdownTextEdit(QWidget *parent)
     connect(this, &QOwnNotesMarkdownTextEdit::zoomOut, this, [this](){
         onZoom(/*in=*/ false);
     });
+    connect(this, &QOwnNotesMarkdownTextEdit::newLinePosition, this, [this](int position){
```

Comment:
since this signal is QON specific, maybe we can just emit it in this class instead of in the library?